W-20092166 publish uc anypoint (#4)
mulesoft/docs-connector-builder: modules/ROOT/pages/releasing-connectors.adoc

== Release Connectors to Anypoint Platform 

> To make your connector available on Anypoint Platform, release it to Anypoint Exchange. Connector Builder supports private releases to Anypoint Exchange only. The connector can still be shared with other accounts or teams. To release a connector to Anypoint Exchange, see xref:exchange::to-publish-assets-maven.adoc[]. When executing `mvn deploy`, ensure you are in the `{projectname}-anypoint-connector-model` directory.

Reviewer: ```suggestion
To make your connector available on Anypoint Platform, release it to Anypoint Exchange. Connector Builder supports private releases to Anypoint Exchange only. The connector can still be shared with other accounts or teams. To release a connector to Anypoint Exchange, see xref:exchange::to-publish-assets-maven.adoc[]. Run `mvn deploy` from the `{projectname}-anypoint-connector-model` directory.
```
